terraform/shared/shared.tf.json.template.py

```diff
@@ -53,30 +53,14 @@ def paren(s: str) -> str:
                         # Filtering on the errorCode alone catches too many false
                         # positives, so we exclude those logs with the additional
                         # conditions below.
-                        paren(' || '.join([
-                            '$.eventSource != "s3.amazonaws.com"',
-                            '$.eventName != "GetObject"',
-                            '$.userIdentity.invokedBy != "cloudfront.amazonaws.com"',
-                        ])),
-                        paren(' || '.join([
-                            '$.eventSource != "s3.amazonaws.com"',
-                            '$.eventName != "HeadBucket"',
-                            '$.userIdentity.invokedBy != "config.amazonaws.com"',
-                        ])),
-                        paren(' || '.join([
-                            '$.eventSource != "s3.amazonaws.com"',
-                            '$.eventName != "HeadObject"',
-                            '$.userIdentity.invokedBy != "cloudfront.amazonaws.com"',
-                        ])),
+                        # Note: Contrary to CloudWatch queries, a metric filter
+                        # pattern such as "$.foo != 'bar'" will not match a
+                        # record that does not contain the field "foo", nor is
+                        # there a way to match absense of a field.
                         paren(' || '.join([
                             '$.eventSource != "inspector2.amazonaws.com"',
                             '$.eventName != "DescribeOrganizationConfiguration"',
                         ])),
```

Comment:
The comment above indicates that this is ineffective/broken. Why did you leave it in?

My [comment on the issue](https://github.com/DataBiosphere/azul/issues/5884#issuecomment-2008265523) meant to request the removal of all exception clauses so that the filter matches the SecurityHub control again.

Reply:
I thought it'd be desirable to leave these filter exclusions in since the fields `eventSource` and `eventName` are always included in the trail logs, and therefore can safely be filtered against with a `!=`.
The fixup commit removes these remaining exclusions so the filter now matches the SecurityHub control.